README.md

```diff
@@ -1,69 +1,67 @@
 <p align="center">
     <a href="https://github.com/yiisoft" target="_blank">
-        <img src="https://yiisoft.github.io/docs/images/yii_logo.svg" height="80px">
+        <img src="https://yiisoft.github.io/docs/images/yii_logo.svg" height="80px" alt="Yii">
     </a>
     <a href="https://www.mysql.com/" target="_blank">
-        <img src="https://labs.mysql.com/common/logos/mysql-logo.svg" height="80px">
+        <img src="https://labs.mysql.com/common/logos/mysql-logo.svg" height="80px" alt="MySQL">
     </a>
-    <h1 align="center">MySQL and MariaDB driver for Yii Database</h1>
+    <a href="https://www.mariadb.com/" target="_blank">
+        <img src="https://mariadb.com/wp-content/uploads/2019/11/mariadb-logo-vertical_blue.svg" height="80px" alt="MariaDB">
```

Comment:
It looks strange.

![image](https://github.com/yiisoft/db-mysql/assets/525501/a5e3911b-6278-4c2d-85b4-265a5634dfbd)

Reply:
Remove logo mariadb... It gets better?

Reply:
done